src/codeu/chat/common/Interests.java

```diff
@@ -0,0 +1,52 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collection;
+
+import codeu.chat.util.Serializer;
+import codeu.chat.util.Serializers;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public final class Interests {
+
+  public static final Serializer<Interests> SERIALIZER = new Serializer<Interests>() {
+
+    @Override
+    public void write(OutputStream out, Interests value) throws IOException {
+
+      Serializers.collection(Uuid.SERIALIZER).write(out, value.interests);
+      Time.SERIALIZER.write(out, value.lastStatusUpdate);
+      Time.SERIALIZER.write(out, value.creation);
+
+    }
+
+    @Override
+    public Interests read(InputStream in) throws IOException {
+
+      return new Interests(
+          Serializers.collection(Uuid.SERIALIZER).read(in),
+          Uuid.SERIALIZER.read(in),
+          Time.SERIALIZER.read(in),
+          Time.SERIALIZER.read(in)
+      );
+
+    }
+  };
+
+  public final Collection<Uuid> interests;
+  public final Uuid id;
```

Comment:
Ah, OK. The nice part is that @lysanderp41 pointed out today that we don't need the InterestsContext, and this field is used to create an InterestsContext. If that's the only thing this field is used for, then this field can go away as well.

I feel a bit bad lumping this in with the deletion work because it's not as simple as the rest of it: Deleting this field means we no longer need to set this field, which means we no longer need to make it a constructor argument for this class, which means we need to change the code that constructs this class. We might also need to change the transaction log format and transaction log parsing code.

Still, we should probably do it, because having this field here just to support the creation of a class that we don't need is silly. @mlk209 , would you be able to help Lysander out with this, maybe by taking over with removing `Interests.id` once Lysander has deleted `InterestsContext`?